### PR TITLE
Add an example of custom icon with Markers

### PR DESCRIPTION
**Pull request by shimatar0**
issue: #30

### CODE REVIEW

File: src/marker.rs [hunk 1]
```diff
@@ -26,8 +26,13 @@ pub struct MarkerOptions {
 }
 
 impl MarkerOptions {
-    pub fn build(&self) -> JsValue {
-        serde_wasm_bindgen::to_value(&self).unwrap()
+    pub fn build(mut self) -> JsValue {
+        let obj: js_sys::Object = serde_wasm_bindgen::to_value(&self).unwrap().into();
+        if let Some(element) = self.element.take() {
+            js_sys::Object::define_property(&obj, &JsValue::from_str("element"), &*element).into()
```
[the remark this answers is not in the document]
Comment (yukinarit) [reply src/marker.rs:32]: Ah sorry, the 3rd property seems to be not a value but [a descriptor object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty). I don't know how to create a descriptor object correctly.

Can you please try [js_sys::Reflect::set](https://docs.rs/js-sys/latest/js_sys/Reflect/fn.set.html) instead? 🤔

[the remark this answers is not in the document]
Comment (shimatar0) [reply src/marker.rs]: @yukinarit 
Thanks, Your advice is great!
Looks like I got the display right when I used [js_sys::Reflect::set](https://docs.rs/js-sys/latest/js_sys/Reflect/fn.set.html). I'll proceed😊

[the remark this answers is not in the document]
Comment (shimatar0) [reply src/marker.rs]: @yukinarit 
Done! Thanks a lot for the advice😊

**Review by yukinarit — APPROVED** [review]
Awesome! 🎉 

Screen Shot 2023-04-25 at 11 44 19

**Pull request merged**